lib/plausible_web/live/funnel_settings/form.ex

```diff
@@ -39,119 +39,109 @@ defmodule PlausibleWeb.Live.FunnelSettings.Form do
+      phx-window-keydown="cancel-add-funnel"
+      phx-key="Escape"
+    >
+    </div>
+    <div class="fixed inset-0 flex items-center justify-center mt-16 z-50 overlofw-y-auto overflow-x-hidden">
+      <div class="w-2/5 h-full">
+        <div id="funnel-form">
+          <.form
+            :let={f}
+            for={@form}
+            phx-change="validate"
+            phx-submit="save"
+            phx-target="#funnel-form"
+            phx-click-away="cancel-add-funnel"
+            onkeydown="return event.key != 'Enter';"
+            class="bg-white dark:bg-gray-800 shadow-md rounded px-8 pt-6 pb-8 mb-4 mt-8"
           >
-            Funnel Name
-          </label>
-          <.input field={f[:name]} />
+            <h2 class="text-xl font-black dark:text-gray-100 mb-6">Add Funnel</h2>
 
-          <div id="steps-builder">
-            <label class="mt-6 block text-sm font-medium text-gray-700 dark:text-gray-300 mb-2">
-              Funnel Steps
+            <label for={f[:name].name} class="block mb-3 font-medium dark:text-gray-100">
+              Funnel Name
             </label>
 
-            <div :for={step_idx <- @step_ids} class="flex mb-3">
-              <div class="w-2/5 flex-1">
-                <.live_component
-                  submit_name="funnel[steps][][goal_id]"
-                  module={PlausibleWeb.Live.Components.ComboBox}
-                  suggest_fun={&PlausibleWeb.Live.Components.ComboBox.StaticSearch.suggest/2}
-                  id={"step-#{step_idx}"}
-                  options={reject_alrady_selected("step-#{step_idx}", @goals, @selections_made)}
-                />
-              </div>
-
-              <div class="w-min inline-flex items-center align-middle">
-                <.remove_step_button :if={length(@step_ids) > Funnel.min_steps()} step_idx={step_idx} />
+            <.input
+              field={f[:name]}
+              phx-debounce={200}
+              autocomplete="off"
+              placeholder="e.g. From Blog to Purchase"
+              autofocus
+              class="w-full focus:ring-indigo-500 focus:border-indigo-500 dark:bg-gray-900 dark:text-gray-300 block w-7/12 rounded-md sm:text-sm border-gray-300 dark:border-gray-500 w-full p-2 mt-2"
+            />
+
+            <div id="steps-builder" class="mt-6">
+              <label class="font-medium dark:text-gray-100">
+                Funnel Steps
+              </label>
+
+              <div :for={step_idx <- @step_ids} class="flex mb-3 mt-3">
+                <div class="w-2/5 flex-1">
+                  <.live_component
+                    submit_name="funnel[steps][][goal_id]"
+                    module={PlausibleWeb.Live.Components.ComboBox}
+                    suggest_fun={&PlausibleWeb.Live.Components.ComboBox.StaticSearch.suggest/2}
+                    id={"step-#{step_idx}"}
+                    options={reject_alrady_selected("step-#{step_idx}", @goals, @selections_made)}
```

Comment:
```suggestion
                    options={reject_already_selected("step-#{step_idx}", @goals, @selections_made)}
```

perhaps a good opportunity to fix that typo here an in the original definition below? 😄

Reply:
done, thanks